yaml/roads.yaml

```diff
@@ -32,6 +31,14 @@ global:
       sport: {col: sport}
       man_made: {col: man_made}
       walking_network: {expr: "mz_hiking_network(osm_id)"}
+      bicycle_network: {expr: "mz_cycling_network(tags, osm_id)"}
+      oneway: {col: oneway}
```

Comment:
For the `oneway` column? I only just moved that so both `oneway` and `oneway_bicycle` were next to each other. so this won't change anything wrt to `oneway`.

But, we do have some logic to normalize the `oneway` tag, which we don't do for `oneway:bicycle`. Do we want to do something similar for `oneway:bicycle` too?